src/api.js

```diff
@@ -21,3 +21,11 @@ export const getArticleComments = (article_id) => {
     return data.comments;
   });
 };
+
+export const updateArticleVotes = (article_id, newVote) => {
+  return api
+    .patch(`/articles/${article_id}`, { inc_votes: newVote })
+    .then(({ data }) => {
+      return data;
+    });
+};
```

Comment:
Nice, but you are not making use of the data that comes back so there isnt really a need to have this .then(...) logic here either